startup/buckets.json

```diff
@@ -2,11 +2,11 @@
   {
     "name": "default",
     "ramQuotaMB": 100,
-    "bucketType": "membase",
+    "bucketType": "couchbase",
```

Comment:
Did you test this change against our current minimum version of 5.0.1? It appears that "couchbase" was a supported value in 5.0.1 based on docs, and "membase" deprecated, but I want to make sure we've confirmed with a test.

Reply:
I did not test this. I was going by the documentation for Couchbase 5.0 found here: https://docs.couchbase.com/server/5.0/cli/cbcli/couchbase-cli-bucket-create.html#options and here https://docs.couchbase.com/server/5.0/rest-api/rest-bucket-create.html#http-method-and-uri
Both of these pages show that `couchbase` is supported and also mention that `memcached` is supported indicating that `membase` isn't even the proper text to be using here.

Reply:
I've tested it, it's fine. The documentation does, in fact, match reality. I did find some other issues with old versions I need to look at, though, probably going to deprecate 5.0 and 5.1.